### PR TITLE
Containers

### PR DESCRIPTION
**Pull request by chrisbolin**
Implements VictoryContainers for #75

### CODE REVIEW

**Comment by chrisbolin** [comment]
Here's what the four containers look like...
apr-13-2017 19-37-37

**Comment by chrisbolin** [comment]
(lint failed. i'll fix that)

**Comment by boygirl** [comment]
@chrisbolin awesome!

It looks like there is quite a bit of duplicated code between the native and web versions. Could this be refactored to share as much as possible so we keep a smaller maintenance surface?

**Comment by chrisbolin** [comment]
@boygirl yup! i have a list of ways we can consolidate. it won't be exactly trivial. i was planning on opening an issue after this one to "dedup" the codebases. does that work?

**Comment by chrisbolin** [comment]
just pushed a fix for `createContainer()` on native (and included a demo)

**Comment by boygirl** [comment]
@chrisbolin yes, that works for me.

File: lib/components/victory-brush-container.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { Rect, G } from "react-native-svg";
+import { last } from "lodash";
```
Comment (boygirl) [inline lib/components/victory-brush-container.js]: You should change how we're depending on `lodash` in `package.json`. I think prior to this pr we were only depending on `lodash.omit`

File: lib/components/victory-container.js [hunk 2]
```diff
@@ -14,12 +79,14 @@ export default class extends VictoryContainer {
     const parentProps = Object.assign({className}, nativeStyle, svgProps);
     return standalone !== false ?
       (
-        <Svg {...parentProps}>
-          {title ? <title id="title">{title}</title> : null}
-          {desc ? <desc id="desc">{desc}</desc> : null}
-          {children}
-          <Portal ref={this.savePortalRef}/>
-        </Svg>
+        <View {...this.panResponder.panHandlers}>
```
Comment (boygirl) [inline lib/components/victory-container.js]: This makes it pretty definite that container components with events can not be used with custom `<svg>` at least in victory-native. I'm fine with this constraint. I'll open an issue to document it.

File: lib/helpers/native-helpers.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import omit from "lodash.omit";
```
Comment (boygirl) [inline lib/helpers/native-helpers.js]: standardize lodash import

**Comment by chrisbolin** [comment]
all that makes sense @boygirl - thanks for taking a look!

**Comment by chrisbolin** [comment]
after we refactor victory-chart and victory-core to reduce duplication, we can revisit if we need the full lodash dependency

**Pull request merged**